dev/devicelab/manifest.yaml

```diff
@@ -468,6 +468,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["linux/android"]
 
+  flutter_test__watch_test:
+    description: >
+      Ensures that --watch flag to flutter test command works as intended.
+    stage: devicelab
+    required_agent_capabilities: ["linux/android"]
```

Comment:
Mark this as flaky

Reply:
Has this been observed to be flaky?  I have no issue adding it, but I would like to see if I could address the flakiness if so.  All the failures below seems to be a cancellation or an analyzer error(trailing whitespace).

Reply:
All new devicelab tests start with a flaky flag

Reply:
Good to know

Reply:
Done